src/main/io/gps.c

```diff
@@ -789,6 +797,20 @@ static bool gpsNewFrameNMEA(char c)
                         *gpsPacketLogChar = LOG_NMEA_RMC;
                         gpsSol.groundSpeed = gps_Msg.speed;
                         gpsSol.groundCourse = gps_Msg.ground_course;
+                        #ifdef USE_RTC_TIME
```

Comment:
Please put preprocessor directives at the beginning of the line - this makes the code more readable.